extensions/wikia/AchievementsII/specials/templates/SpecialCustomize.tmpl.php

```diff
@@ -1,180 +1,199 @@
+								(<?= wfMessage( 'achievements-points', $config->getLevelScore( $badge->getLevel() ) )->escaped(); ?>
+								)</p>
+							<p><?= $badge->getGiveFor(); ?></p>
+						</div>
 
-					<div class="image-form">
-						<p>
-							<img width="90" height="90" src="<?=$badge->getPictureUrl(90);?>" />
-							<br />
+						<div class="image-form">
+							<p>
+								<img width="90" height="90" src="<?= $badge->getPictureUrl( 90 ); ?>"/>
+								<br/>
 							<span class="custom-text"><?= wfMessage( 'achievements-customize' )->escaped(); ?>
-								<br />
-								<a href="#" onclick="Achievements.revert(this, <?=$badgeTypeId;?>, <?=$lap;?>); return false;">
+								<br/>
+								<a href="#"
+								   onclick="Achievements.revert(this, <?= $badgeTypeId; ?>, <?= $lap; ?>); return false;">
 									<?= wfMessage( 'achievements-revert' )->escaped(); ?>
 								</a>
 							</span>
-						</p>
-						<form method="POST" enctype="multipart/form-data" class="customize-upload" action="<?= $wgScriptPath ?>/index.php?action=ajax&amp;rs=AchAjax&amp;method=uploadBadgeImage&amp;type_id=<?=$badgeTypeId?>&amp;lap=<?=$lap?>&amp;level=<?=$badge->getLevel();?>" onsubmit="return Achievements.submitPicture(this);">
-							<p class="input">
-								<input name="wpUploadFile" type="file"/>
-								<button type="submit"><?= wfMessage( 'achievements-send' )->escaped(); ?></button>
 							</p>
-						</form>
-					</div>
-				</li>
-			<?endforeach;?>
-		</ul>
-	</div>
-<?endforeach;?>
+							<form method="POST" enctype="multipart/form-data" class="customize-upload"
+								  action="<?= $wgScriptPath ?>/index.php?action=ajax&amp;rs=AchAjax&amp;method=uploadBadgeImage&amp;type_id=<?= $badgeTypeId ?>&amp;lap=<?= $lap ?>&amp;level=<?= $badge->getLevel(); ?>"
+								  onsubmit="return Achievements.submitPicture(this);">
+								<p class="input">
+									<input name="wpUploadFile" type="file"/>
+									<button type="submit"><?= wfMessage( 'achievements-send' )->escaped(); ?></button>
+								</p>
+							</form>
+						</div>
+					</li>
+				<? endforeach; ?>
+			</ul>
+		</div>
+	<? endforeach; ?>
 
-<?php
-global $wgAchievementsEditOnly;
-$sections = array();
-if ( empty( $wgAchievementsEditOnly ) ) {
-	$sections = array(
-		'special' => array(),
-		'secret' => array()
-	);
-	foreach( $config->getNotInTrackStatic() as $badgeTypeId => $badgeData ) {
-		$section = null;
-		if( $config->isSpecial( $badgeTypeId ) )
-			$section = 'special';
-		elseif( $config->isSecret( $badgeTypeId ) )
-			$section = 'secret';
-		else
-			continue;
+	<?php
+	global $wgAchievementsEditOnly;
+	$sections = array();
+	if( empty( $wgAchievementsEditOnly ) ) {
+		$sections = array(
+			'special' => array(),
+			'secret' => array()
+		);
+		foreach( $config->getNotInTrackStatic() as $badgeTypeId => $badgeData ) {
+			$section = null;
+			if( $config->isSpecial( $badgeTypeId ) )
+				$section = 'special';
+			elseif( $config->isSecret( $badgeTypeId ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)